README.md

```diff
@@ -8,3 +8,11 @@ This is a Spring Boot service to manage data for Mongen clients
 docker run --name mongen_core_db -e POSTGRES_PASSWORD=123123 -e POSTGRES_USER=mongen -e POSTGRES_DB=mongen_db -p 5432:5432 -d postgres:12
 docker exec -it mongen_core_db psql -h localhost -p 5432 -U mongen mongen_db
 ```
+
+#### Then start the database at any time with
+
+```
+docker start mongen_core_db
```

Comment:
and the core?

Reply:
For the core is just importing the project in IntelliJ or Eclipse and start it, it should be able to detect the db and initialize everything

Reply:
aaa, it's not dockerized yet, okok